config/puma.rb

```diff
@@ -1,20 +1,23 @@
 # Puma can serve each request in a thread from an internal thread pool.
-# The `threads` method setting takes two numbers a minimum and maximum.
+# The `threads` method setting takes two numbers: a minimum and maximum.
 # Any libraries that use thread pools should be configured to match
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
-# and maximum, this matches the default thread size of Active Record.
+# and maximum; this matches the default thread size of Active Record.
 #
-threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i
+threads_count = ENV.fetch("RAILS_MAX_THREADS") { 20 }
```

Comment:
I probably spend half of my day sniffing glue, but wasn't the sheldon article related to the AR connection pool config?

I have no idea what type of load this service receives in production. perhaps we need to set these sorts of options based on how the passenger <-> httpd web server is configured now.

I suggest that we reverse this RAILS_MAX_THREADS value back to 5 and add an issue to compare the current production passenger config to the puma config.

thoughts?

Reply:
I'm perfectly cool to dial it back to 5.  That's the way in which David still leaves it for default Rails projects.